spark/src/main/java/org/opensearch/sql/spark/flint/FlintIndexType.java

```diff
@@ -12,24 +12,24 @@
 /** Enum for FlintIndex Type. */
 public enum FlintIndexType {
   SKIPPING("skipping_index"),
-  COVERING("covering_index"),
+  COVERING("index"),
   MATERIALIZED("materialized_view");
 
-  private final String name;
+  private final String suffix;
   private static final Map<String, FlintIndexType> ENUM_MAP;
 
-  FlintIndexType(String name) {
-    this.name = name;
+  FlintIndexType(String suffix) {
+    this.suffix = suffix;
   }
 
-  public String getName() {
-    return this.name;
+  public String getSuffix() {
+    return this.suffix;
   }
 
   static {
     Map<String, FlintIndexType> map = new HashMap<>();
     for (FlintIndexType instance : FlintIndexType.values()) {
-      map.put(instance.getName().toLowerCase(), instance);
+      map.put(instance.getSuffix().toLowerCase(), instance);
```

Comment:
why is the `.toLowerCase()` necessary ?

Reply:
````
  static {
    Map<String, FlintIndexType> map = new HashMap<>();
    for (FlintIndexType instance : FlintIndexType.values()) {
      map.put(instance.getSuffix().toLowerCase(), instance);
    }
    ENUM_MAP = Collections.unmodifiableMap(map);
  }
 
  

  public static FlintIndexType get(String name) {
    return ENUM_MAP.get(name.toLowerCase());
  }
```
Both of these are in sync.

Reply:
actually we don't need this method
will remove it.